en/Alert.md

```diff
@@ -0,0 +1,22 @@
+### Alert
+
+General guidelines for alerts and trips:
+* If a TripDescriptor is specified in an alert, the informed_entities must provide a `trip_id`. 
+* When a TripDescriptor doesn’t specify `start_date` or `start_time`, all trips with the corresponding `trip_id` are affected.
+* Specifying start_time for frequency-based trips (defined in GTFS `frequencies.txt`) doesn’t define a trip instance. However, it allows you to attach alerts to the corresponding trip defined in trip updates, even if these trip updates are in a separate feed.  
+* Frequency-based trips (defined in GTFS `frequencies.txt`) with unspecified start_time show the alert for all route results involving the trip.
+* When `trip_id` and `start_time` are within `exact_time=1` interval, `start_time` must be later than the beginning of the interval by an exact multiple of `headway_secs`. 

```

Comment:
"must" indicates this should be in the spec.

Reply:
Addressed in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/commit/26dd2eb173316bcf3caaaf93a9a0c9481103bc42.